src/tests/ftest/util/soak_utils.py

```diff
@@ -585,27 +583,22 @@ def launch_vmd_identify_check(self, name, results, args):
     else:
         uuids = uuid_list
     self.log.info("VMD device UUIDs: %s", uuids)
-    host_uuids = get_storage_query_device_uuids(self.dmg_command)
-    for host, uuid_dict in host_uuids.items():
-        uuid_list = sorted(uuid_dict.keys())
-        self.log.info("Devices on host %s: %s", host, uuid_list)
-        # Now check whether the random uuid belongs to a particular host.
-        for uuid in uuids:
-            if uuid in uuid_list:
-                dmg.hostlist = host
-                # Blink led
-                dmg.storage_led_identify(ids=uuid, timeout=2)
-                # check if led is blinking
-                result = dmg.storage_led_check(ids=uuid)
-                # determine if leds are blinking as expected
-                for value in list(result['response']['host_storage_map'].values()):
-                    if value['storage']['smd_info']['devices']:
-                        for device in value['storage']['smd_info']['devices']:
-                            if device['ctrlr']['led_state'] != "QUICK_BLINK":
-                                failing_vmd.append([device['ctrlr']['pci_addr'], value['hosts']])
-                                status = False
-                            # reset leds to previous state
-                            dmg.storage_led_identify(ids=uuid, reset=True)
+
+    for uuid in uuids:
+        # Blink led
+        self.dmg_command.storage_led_identify(ids=uuid, timeout=2)
```

Comment:
The previous code issued this command on a subset of the server hosts (via its hostlist) and this new code is issuing the command to all hosts.  Is this intentional?

Reply:
In the current config for soak on aurora this is not an issue because soak is only run on 128 servers.  But if this goes beyond the current config then we should limit the number of vmd devices to check.

Reply:
@phender I talked with @mjean308 and the real fix will go into the release/2.6 branch eventually. This is just to move forward on Aurora now